### PR TITLE
Add Setup Mailserver with Mailcow

### PR DESCRIPTION
**Pull request by fbrettnich**
Here is a tutorial about "Learn how to setup your own mail server with Mailcow"

---

I have read and understood the Contributor's Certificate of Origin at the end of the template and I hereby certify that I meet the contribution criteria described in it.

Signed-off-by: fbrettnich <github@fbrettnich.de>

### CODE REVIEW

**Comment by killerbees19** [comment]
Your directory name starts with a space: `--> <--setup-mailserver-with-mailcow` 😉

Maybe it's a good idea to rename it at your branch…

File: community-tutorials/setup-mailserver-with-mailcow/01-en.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+netcup_product_url: https://www.netcup.de/bestellen/produkt.php?produkt=2554
+language: en
+available_languages: [en, de]
+---
+
+# Introduction
+In this tutorial you will learn how to set up and run your own mail server using Mailcow.  
+We use [mailcow dockerized](https://github.com/mailcow/mailcow-dockerized) for this.
+
+This tutorial is based on the  [Mailcow Documentation](https://mailcow.github.io/mailcow-dockerized-docs/i_u_m_install/).
```
Comment (fbrettnich) [inline community-tutorials/setup-mailserver-with-mailcow/01-en.md:20]:
```suggestion
This tutorial is based on the [Mailcow Documentation](https://mailcow.github.io/mailcow-dockerized-docs/i_u_m_install/).
```

**Comment by raphi-0901** [comment]
Hey, the requirements of the tutorial are too high. how do you connect your root server to a domain? You could describe this in an extra tutorial and then link it in this tutorial.

**Comment by fbrettnich** [comment]
> Hey, the requirements of the tutorial are too high. how do you connect your root server to a domain? You could describe this in an extra tutorial and then link it in this tutorial.

Hi, this is described in step 1.1
I currently do not have a domain at netcup to create an extra tutorial for it

https://github.com/fbrettnich/community-tutorials/blob/main/community-tutorials/setup-mailserver-with-mailcow/01-en.md#step-11---preparation-domain-and-hostname

**Comment by raphi-0901** [comment]
Hey, sorry I was a little bit confused but I got it now.

**Comment by netcupClaudiaM** [comment]
Hi fbrettnich, thank you for contributing! It's been a while since you opened this PR and it probably is an open secret we're currently overwhelmed with reviewing all the awesome tutorials we got. Please be patient - we're about to publish tutorials this and next week. Thank you!

**Comment by raphi-0901** [comment]
Hey @fbrettnich, could you please upload this picture `netcup-rdns.png` in english so that everything is consistent? Thanks!

**Comment by fbrettnich** [comment]
> Hey @fbrettnich, could you please upload this picture `netcup-rdns.png` in english so that everything is consistent? Thanks!

Hi @raphi-0901,
I have now uploaded a picture of the English version.
_(GitHub may still have the old version in its cache right now)_

Thanks

**Pull request merged**